bin.src/validateDrp.py

```diff
@@ -29,37 +30,46 @@
 
 
 if __name__ == "__main__":
-    helpMessage = """Usage: validateDrp.py repo configFile
+    description="""
+    Calculate and plot validation Key Project Metrics from the LSST SRD.
 
-    Arguments
-    ---------
-    `repo` : str
-        path to a repository containing the output of processCcd
-    `configFile` : str
-        YAML configuration file declaring the parameters for this run.
-
-    Output
-    ------
+    Produces results to:
     STDOUT
         Summary of key metrics
     *.png
         Plots of key metrics.  Generated in current working directory.
+    *.json
```

Comment:
If it were possible (I've not looked at the rest of the code yet), it would be helpful to list exactly which files are being produced rather than just a glob -- makes it easier to clean up again afterwards without potentially clobbering my own files in the directory. Not a big deal, though.